recipes/pymagicc/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pymagicc" %}
+{% set version = "2.0.0b11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256:
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

Comment:
I think the build error occurs for Python 3.8, so can you please try with `python >=3.5,<3.8` in both `host` and `run` sections?

Reply:
I haven't filled in the sha256 bit yet (cause we need to do an sdist release on pypi), that's probably it right?

Reply:
Ah, yes, you are correct. You can use the release tarball from GitHub it you want, to test your recipe until the PyPi tarball is uploaded.